tracer/samples/WindowsContainer/docker-compose.yml

```diff
@@ -13,8 +13,9 @@ services:
       ASPNETCORE_ENVIRONMENT: Production
 
   datadog-agent:
-    image: "gcr.io/datadoghq/agent:7"
+    image: "gcr.io/datadoghq/agent:latest"
     environment:
+      DD_HOSTNAME: "windowscontainer-datadog-agent"
```

Comment:
These is the "real" issue - it's a recent requirement in the latest version of the agent. We should ensure that all other docker-compose files that use it similarly have a hostname, e.g.: https://github.com/DataDog/dd-trace-dotnet/blob/a72b74fc8e67d8d8a6430628fe8643b3a693d2bc/tracer/samples/NugetDeployment/docker-compose.yml#L7